api/Model/IGenericCrudModel.cs

```diff
@@ -0,0 +1,9 @@
+using System.ComponentModel.DataAnnotations;
```

Comment:
Ik denk dat deze too much is, Generic Controllers en Services zijn wel goed!

Reply:
Moet wel, want ik moet garanderen dat er een Id in de entity voor komt. Dat kan met Generic Constraints